Katello/katello
# Fixes #33000 - Display checkbox/select column for non-admin users

- Use `hasUpstreamConnection` instead of `content_disconnected` from `Setting` to determine if checkbox/select column should be enabled.
- Remove all `content_disconnected` related code since it's only being used by Subscription. 

`view_settings` permission is required to retrieve `content_disconnected` from `Setting`.  A non-admin user may not have this permission.
`hasUpstreamConnection` is set by `pingUpstreamSubscriptions` which does not require a permission.

### Steps to Test
- Create a role and assign 'Organization', 'Location' and 'Subscription' permissions.
- Associate the role with a user. 
- Login as this user and navigate to 'Subscriptions' page. Verify the checkboxes next to each subscription are displayed.
